externals/libcurl/src/configureHook.sh

```diff
@@ -6,6 +6,15 @@ cd $cares_location
 sh makeHook.sh
 cd $cdir
 
+curl_ssl_config="--with-ssl"
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
The `[[ ... ]]` is not a "bashism", is it?  In other places, we used

    case $(uname) in
      Darwin )
      ...
      ;;
    esac

but I'm not sure if it makes a difference